app/controllers/thredded/messageboard_groups_controller.rb

```diff
@@ -20,6 +20,14 @@ def create
       end
     end
 
+    def show
+      @groups = Thredded::MessageboardGroupView.grouped(
+        policy_scope(Thredded::Messageboard.where(group: params[:id])),
+        user: thredded_current_user
+      )
+      render template: 'thredded/messageboards/index'
```

Comment:
Let's use a separate template. The page should have a different title and view hooks.